complexify-android/src/main/java/co/infinum/complexify/ui/EditTextExt.kt

```diff
@@ -0,0 +1,21 @@
+package co.infinum.complexify.ui
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.EditText
+import co.infinum.complexify.Complexify
+import co.infinum.complexify.ComplexityListener
+
+fun EditText.setComplexityListener(complexify: Complexify, complexityListener: ComplexityListener) {
```

Comment:
Can we add a JVM name to this so the result wouldn't be some ugly Kotlin generated name but something like ComplexifyKTX or similar. Not my best naming either.

Reply:
For now, I settled to leave it the same way as KTX does it